### ISSUE TITLE
Issue/Question: Clear dock and run cell with one keystroke

### ISSUE THREAD

**Issue by dsanalytics** [nteract/hydrogen#1066]
Re: Issue/Question: Clear dock and run cell with one keystroke

I'm new to Atom and Hydrogen - evaluating switching to it from jupyter notebook and visual studio code. Great work BTW - thanks for all your time put into it!

Question/issue: I prefer to see the output in the dock, but it appears that hydrogen does not offer automatic option to clear the dock between cell runs - right now it just keeps adding them at end. If so, can someone show how to map a single shortcut in Atom to two hydrogen commands to be executed in sequence - namely: 
'hydrogen:clear-results' & 'hydrogen:run-cell'? 
In other words I'd like be able to press ctrl-alt-enter and make hydrogen clear the dock and run the cell.

I've checked http://flight-manual.atom.io/behind-atom/sections/keymaps-in-depth/ but it shows only how to combine built-in editor ones.

Thanks in advance.
 
Windows 10 Home Premium 64bit 
Atom 64bit v 1.21.1
Hydrogen 2.0.1

**Comment by BenRussert** [comment]
Your idea in #1067 will resolve this. Once that is added a mapping in your `keymap.cson` to `hydrogen:clear-results` should work nicely.

Closing in favor of #1067
